sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
 ## 1.4.0b2 (Unreleased)
+- All `get_token` methods consistently require at least one scope argument,
+raising an error when none is passed. Although `get_token()` may sometimes
+have succeeded in prior versions, it couldn't do so consistently because its
+behavior was undefined, and dependened on the credential's type and internal
+state. ([#10366](https://github.com/Azure/azure-sdk-for-python/pull/10366))
```

Comment:
Can we link to the/an issue instead? The issue can be marked as resolved by this PR.

Reply:
I linked the PR because its description focuses on the rationale behind this change and links to the motivating issue as an example of what goes wrong without it. I think that issue's more narrow focus on a specific Graph + Key Vault scenario makes it a worse place to start for someone wanting more information about this change.

But if you aren't persuaded sure, we can link to the issue 😸

Reply:
I generally prefer to link to issues since they are inherently talking about a single issue. Pointing to the PR gets a bit recursive/meta since we are discussing how to document the change in the PR comments :). 

If you feel like the issue is too specific, feel free to open a new issue that outlines the specific context.